draft-ietf-quic-load-balancers.md

```diff
@@ -991,12 +991,15 @@ Shared-State Retry Service Token {
 
 The tokens are protected using AES128-GCM as follows:
 
-* The token key and IV are retrieved using the Key Sequence.
+* Key Sequence is the 8 bit identifier to retrieve The token key and AEAD IV.
 
-* The nonce, N, is formed by combining the IV with the 96 bit unique token
-number. The 96 bits of the unique token number are left-padded with zeros to the
-size of the IV. The exclusive OR of the padded unique token number and the IV
-forms the AEAD nonce.
+* The AEAD IV, is a 96 bit sequence number maintained by configuration agent,
```

Comment:
It's not a sequence number and it's not incremented by one. The IV is a product of the key derivation function.

Reply:
I have rewrite the illustration of `AEAD IV`, please check it ;-)